ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/eventstream/converters/EventStreamHttpConverter.java

```diff
@@ -39,7 +41,7 @@ public boolean canWrite(Class<?> clazz, MediaType mediaType) {
 
 	@Override
 	public List<MediaType> getSupportedMediaTypes() {
-		return List.of(MediaType.ALL);
+		return List.of(DEFAULT_MEDIA_TYPE, MediaType.ALL);
```

Comment:
Should this be needed? Since MediaType.ALL should contain "all" no ? :)

Reply:
This is a Spring Boot issue. If no content-type is given, then MediaType.ALL uses application/json by default. In our case we want to use text/turtle as default. Hence the explicit DEFAULT_MEDIA_TYPE.